driver-core/src/main/com/mongodb/internal/operation/AsyncQueryBatchCursor.java

```diff
@@ -69,15 +69,18 @@
     private volatile int batchSize;
     private final AtomicInteger count = new AtomicInteger();
     private volatile BsonDocument postBatchResumeToken;
-    private volatile BsonTimestamp operationTime;
-    private volatile boolean firstBatchEmpty;
-    private volatile int maxWireVersion = 0;
+    private final BsonTimestamp operationTime;
+    /**
+     * May be true for a tailable cursor.
+     */
```

Comment:
It may be true for any type of cursor.  It may be confusing to call it out for tailable cursors only.

Reply:
This comment was originally in the code: https://github.com/mongodb/mongo-java-driver/pull/654/commits/0893c181da2e774cc8236760869b6b3ac140f775#diff-a781a219ff4737fc3cbf85dcbd57ee59970d024d0994cdf390b352afd1c38d20L194

Once I roll back 0893c181da2e774cc8236760869b6b3ac140f775, this comment will return to its original place.

Reply:
Acknowledged